src/main/java/com/nasrpi/home/HomeRepository.java

```diff
@@ -160,4 +169,36 @@ public boolean moveFile(String source, String destination) {
 
 	}
 
+	public UploadFileResponse uploadFile(MultipartFile file, String uploadPath, FileStorageService fileStorageService) {
+
+		String fileName = fileStorageService.storeFile(file, uploadPath);
+
+		String fileDownloadUri = ServletUriComponentsBuilder.fromCurrentContextPath().path("/downloadFile/")
```

Comment:
add "/downloadFile/" in constants class and then use it

Reply:
done